### PR TITLE
Support Jackalope <1.2 and code clean-up

### PR DESCRIPTION
**Pull request by WouterJ**
/cc @dbu

### CODE REVIEW

File: src/Phpunit/DatabaseTestListener.php [hunk 1]
```diff
@@ -110,39 +110,44 @@ private function setUpPhpcrDatabase($suite)
...
 
+        // initialize PHPCR DBAL (new way)
         $process = $this->processBuilder
             ->setArguments(array_merge($this->prefix, array('doctrine:phpcr:init:dbal', '--drop', '--force')))
             ->getProcess();
+
         $process->run();
 
-        while (true) {
-            if ($process->isTerminated()) {
```
Comment (WouterJ) [inline src/Phpunit/DatabaseTestListener.php]: all these while loops and tests were not needed, as `->run()` will always wait until the command is terminated (I probably didn't know this the first time I used the Process component)

**Comment by lsmith77** [comment]
I guess it would make sense to adjust one of the various failing PRs to this branch to verify this before merging.

**Comment by WouterJ** [comment]
good idea! Let's see what happends: https://travis-ci.org/symfony-cmf/TreeBrowserBundle/builds/59158520

**Comment by WouterJ** [comment]
Hmm, I think that means it works :)

**Pull request merged**